packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -1669,7 +1672,9 @@ const VideoPlayerBase = class extends React.Component {
 			infoVisible: showMoreComponents,
 			titleVisible: true,
 			announce: announce < AnnounceState.INFO ? AnnounceState.INFO : AnnounceState.DONE
-		}));
+		}), () => {
+			this.isRenderedAriaProps = true;
```

Comment:
Using `setState` callbacks are generally discouraged, especially when you can achieve the same thing in `componentDidUpdate`. Is there any reason why we can't do this in `cDU`?